### ISSUE TITLE
Extend example with N* spectrum that has spin (angular momenta)

### ISSUE THREAD

**Issue by shenvitor** [ComPWA/gluex-nstar#3]

**Comment by shenvitor** [comment]
This was already included while using `qrules`+`ampform`

**Comment by redeboer** [comment]
That's true, although you will (for now) have to use [ampform-dpd](https://github.com/ComPWA/ampform-dpd) if you have different subsystems and a spinful final state.
